### PR TITLE
Feature/train 877 organism todo list

### PR DESCRIPTION
**Pull request by jhonangelo**
- Create to do list component
- Add option to render kebab menu only when a to do item is not set as done
- Add todo text item onClick 

ISSUE LINK: https://vananaz.atlassian.net/browse/TRAIN-877

### CODE REVIEW

File: src/components/atoms/TodoItem/Component.tsx [hunk 1]
```diff
@@ -26,18 +29,24 @@ const TodoItem = styled.p<Props>`
     `
     text-decoration: line-through;
     color: #9EB7DA;
```
Comment (junrezapico) [inline src/components/atoms/TodoItem/Component.tsx]: I know this isn't part of the pr but have you thought about putting your color values in a colors.ts file? makes it much easier to track and update colors if needed

File: src/components/atoms/TodoItem/Component.tsx [hunk 2]
```diff
@@ -8,6 +8,9 @@ const TodoItem = styled.p<Props>`
   font-size: 14px;
   text-align: left;
   margin: 0;
+  cursor: pointer;
+  border: none;
+  background: none;
 
   ${(props) =>
```
Comment (junrezapico) [inline src/components/atoms/TodoItem/Component.tsx:17]: not urgent for this pr but you might also want to type your prop object here so we could know easier what this object should look like

Comment (jhonangelo) [reply src/components/atoms/TodoItem/Component.tsx:17]: hi can you please elaborate on this one?

Comment (junrezapico) [reply src/components/atoms/TodoItem/Component.tsx:17]: @jhonangelo basically you could make use of typescript to specify what your "props" object should look like in line 15. I forgot where in the doc it is described but here is a [stack overflow link](https://stackoverflow.com/questions/47077210/using-styled-components-with-props-and-typescript) showing what I'm referring to. If you want to use this, then you might also want to look at [transient props](https://styled-components.com/docs/api#transient-props) in styled components as this is the solution to the stack overflow issue

**Review by junrezapico — APPROVED** [review]
LGTM

**Pull request merged**